akka-sample-sharding-scala/src/main/resources/weather-station.conf

```diff
@@ -0,0 +1,23 @@
+akka {
+  loglevel = DEBUG
+
+  http {
+    client.host-connection-pool.max-open-requests = 50
```

Comment:
TODO

Reply:
Tried a quick fix but it just needed to read the response as you said ;) Removed.